modules/helm-understanding-helm.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/getting-started-with-helm-on-openshift.adoc
+
+[id="understanding-helm_{context}"]
+= Understanding Helm
+
+Helm is a CLI tool that simplifies deployment of applications and services to {product-title} clusters. Helm uses a packaging format called charts - a collection of files that describe {product-title} resources. 
```

Comment:
Spell out CLI on first instance:
s/CLI/command-line interface (CLI)
s/charts - a collection of files that describe/charts. A Helm chart is a collection of files that describes/
(See IBM Style Guide - "Dashes" for info about not using en/em dashes in technical info.)